build.gradle.kts

```diff
@@ -97,6 +98,32 @@ allprojects {
 }
 
 subprojects {
+    val jacocoAndroidModules = listOf(
+        "sentry-android-core",
+        "sentry-android-fragment",
+        "sentry-android-navigation",
+        "sentry-android-ndk",
+        "sentry-android-okhttp",
+        "sentry-android-sqlite",
+        "sentry-android-timber",
+        "sentry-compose"
+    )
+    if (jacocoAndroidModules.contains(name)) {
+        afterEvaluate {
+            tasks.getByName("jacocoTestReport") {
```

Comment:
you could also use the default Gradle's `Copy` task, which should be even pretty straightforward https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Copy.html